test/providers/provider_gitlabci.test.js

```diff
@@ -47,6 +47,14 @@ describe('GitLabCI Params', () => {
       service: 'gitlab',
       slug: '',
     }
+    const spawnSync = td.replace(childProcess, 'spawnSync')
```

Comment:
I think we can safely replace this with `execFileSync` in the future, since we are no longer calling both `git` and `hg`